src/coreclr/vm/object.h

```diff
@@ -86,6 +86,10 @@ class ArgDestination;
 
 struct RCW;
 
+#ifdef _DEBUG
```

Comment:
feels odd in object.h. Can it be added to log.h?

Reply:
Ok, done. Actually it just duplicates "_DEBUG" macro. Maybe it should be cleaned up and ifdef's should check "_DEBUG" macro instead?